internal/services/oracle/oracle_autonomous_database_cross_region_disaster_recovery_data_source.go

```diff
@@ -0,0 +1,539 @@
+	PrivateEndpoint                         string   `tfschema:"private_endpoint"`
+	PrivateEndpointIP                       string   `tfschema:"private_endpoint_ip"`
+	PrivateEndpointLabel                    string   `tfschema:"private_endpoint_label"`
+	ProvisionableCPUs                       []int64  `tfschema:"provisionable_cpus"`
+	RemoteDataGuardEnabled                  bool     `tfschema:"remote_data_guard_enabled"`
+	ServiceConsoleUrl                       string   `tfschema:"service_console_url"`
+	SqlWebDeveloperUrl                      string   `tfschema:"sql_web_developer_url"`
+	SubnetId                                string   `tfschema:"subnet_id"`
+	SupportedRegionsToCloneTo               []string `tfschema:"supported_regions_to_clone_to"`
+	TimeCreated                             string   `tfschema:"time_created"`
+	TimeDataGuardRoleChanged                string   `tfschema:"time_data_guard_role_changed"`
+	TimeDeletionOfFreeAutonomousDatabase    string   `tfschema:"time_deletion_of_free_autonomous_database"`
+	TimeLocalDataGuardEnabled               string   `tfschema:"time_local_data_guard_enabled_on"`
+	TimeMaintenanceBegin                    string   `tfschema:"time_maintenance_begin"`
+	TimeMaintenanceEnd                      string   `tfschema:"time_maintenance_end"`
+	TimeOfLastFailover                      string   `tfschema:"time_of_last_failover"`
+	TimeOfLastRefresh                       string   `tfschema:"time_of_last_refresh"`
+	TimeOfLastRefreshPoint                  string   `tfschema:"time_of_last_refresh_point"`
+	TimeOfLastSwitchover                    string   `tfschema:"time_of_last_switchover"`
+	TimeReclamationOfFreeAutonomousDatabase string   `tfschema:"time_reclamation_of_free_autonomous_database"`
+	UsedDataStorageSizeInGbs                int64    `tfschema:"used_data_storage_size_in_gbs"`
+	UsedDataStorageSizeInTbs                int64    `tfschema:"used_data_storage_size_in_tbs"`
+	VnetId                                  string   `tfschema:"virtual_network_id"`
+	AllowedIps                              []string `tfschema:"allowed_ips"`
+}
+
+func (d AutonomousDatabaseCrossRegionDisasterRecoveryDataSource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"resource_group_name": commonschema.ResourceGroupNameForDataSource(),
+
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: validate.AutonomousDatabaseName,
+		},
+	}
+}
+
+func (d AutonomousDatabaseCrossRegionDisasterRecoveryDataSource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"location": commonschema.LocationComputed(),
+
+		// Cross Region Disaster Recovery
+		"remote_disaster_recovery_type": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"replicate_automatic_backups_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		},
+		"database_type": {
```

Comment:
I think we don't need to expose all the readonly fields, maybe just the fields that could be referenced by other resources.